### ISSUE TITLE
Mean and std in config file is resulting in no normalization

### ISSUE THREAD

**Issue by altair199797** [goutamyg/SMAT#13]
In `experiments/mobilevitv2_track/mobilevitv2_256_128x1_ep300.yaml` you have:

```
  MEAN:
  - 0.0
  - 0.0
  - 0.0
  STD:
  - 1.0
  - 1.0
  - 1.0
```

This results in no normalization as `(x - 0)/1 = x` (see [pytorch normalize](https://pytorch.org/vision/main/generated/torchvision.transforms.Normalize.html)).
Correct configuration can be found in [stark original code](https://github.com/researchmm/Stark/blob/main/experiments/stark_st2/baseline.yaml).
Is there a reason for it? Am I missing something?

Kind Regards
Moritz Nottebaum

**Comment by goutamyg** [comment]
Hi,

Thank you for your interest in our work. The normalization parameters in our config file assume that you have already performed ```x = x/255.0```. Most tracking algorithms use ImageNet's normalization parameters (similar to the one you shared above), but [MobileViTv2](https://github.com/apple/ml-cvnets) backbone deployed in our tracking method is pretrained with mean = 0.0 and std = 1.0 (i.e., only normalization applied is ```x = x/255.0```). I have retained these parameters in the config file in case someone wants to try a different MEAN and STD. I hope that answers your question.

**Comment by altair199797** [comment]
Thank you very much for the clarification!
